test/test_gpu3.cpp

```diff
@@ -8567,6 +8567,43 @@ TEST_F(NVFuserTest, AlignedSyncReduction1_CUDA) {
       kernel_string);
 }
 
+// Simple test to check if the aligned block sync is used in aligned
```

Comment:
This is clearly not about aligned block sync 😄

Reply:
..... Yeah, so copy pasted this but forget to change🤦